Jenkinsfile_CNP

```diff
@@ -11,20 +11,26 @@ def product = "sscs-cor"
 def component = "backend"
 
 List<LinkedHashMap<String, Object>> secrets = [
-  secret('sscs-s2s-secret', 'IDAM.S2S-AUTH.TOTP_SECRET'),
+        secret('sscs-s2s-secret', 'IDAM.S2S-AUTH.TOTP_SECRET'),
+        secret('idam-sscs-systemupdate-user', 'IDAM_SSCS_SYSTEMUPDATE_USER'),
+        secret('idam-sscs-systemupdate-password', 'IDAM_SSCS_SYSTEMUPDATE_PASSWORD'),
+        secret('idam-oauth2-client-secret', 'IDAM_OAUTH2_CLIENT_SECRET')
 ]
 
 static LinkedHashMap<String, Object> secret(String secretName, String envVar) {
-  [ $class: 'AzureKeyVaultSecret',
-    secretType: 'Secret',
-    name: secretName,
-    version: '',
-    envVariable: envVar
-  ]
+    [ $class: 'AzureKeyVaultSecret',
+      secretType: 'Secret',
+      name: secretName,
+      version: '',
+      envVariable: envVar
+    ]
 }
 
 withPipeline(type, product, component) {
-    enableSlackNotifications('#sscs-tech')
+    setVaultName('sscs')
```

Comment:
this should be `loadVaultSecrets(secrets)`, I think

Reply:
That's the next line does it matter?

Reply:
You're right. I don't know.